README.md

```diff
@@ -1,24 +1,171 @@
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- A Desktop or Laptop Computer running on with Windows, Mac OS or Linux operating system.
+- Rails installed on the Computer
+
+### Setup
+
+Clone these repositories to your desired folder:
+
+- `https://github.com/GordonOtieno/i-model-cars-backend.git` and 
+- `https://github.com/GordonOtieno/i-model-cars-frontend.git` if you prefer ssh and hit enter.
+- create a directory in your preferred location (e.g in the Desktop or Document directory)
+
+### Install
+
+Install this project with:
+
+- Install Ruby on Rails framework.
+- Use RubyGems as a software package system.
+- Dependencies _(a packet manager is required)_: then you run the command `bundle install` in the terminal
+
+### Usage
+
+To run the project, execute the following command:
+
+- `rails server`
+- This will run the project in the terminal in the specified directory
+
+### Run tests
+
+To run tests, run the following command:
+
+- `rspec spec`
+- This will run all tests on your local machine
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="author"></a>
+
+👤 **Okechukwu Muokwugwo**
+
+- GitHub: [@githubhandle](https://github.com/Okechukwu-muokwugwo)
+- Twitter: [@twitterhandle](https://twitter.com/excel4eva)
+- LinkedIn: [LinkedIn](LinkedIn.com/in/okeimuokwugwo)
+
+👤 **Gordon Otieno**
+
+- GitHub: [@GordonOtieno](https://github.com/GordonOtieno)
+- Twitter: [@GordonOtieno](https://twitter.com/GordonO34459259/twitterhandle)
+- LinkedIn: [GordonOtieno](https://www.linkedin.com/in/gordonotieno/)
+
+👤 **Amine**
+
+- GitHub: [@ouasamine](https://github.com/ouasamine)
+- Twitter: [@ouass_amine](https://twitter.com/ouass_amine)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/amine-ouassef)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> If you like this project then kindly give me a star ⭐️ in the project repository.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments & Attribution <a name="acknowledgements"></a>
+
+- I would like to thank *Gregoire Vella on Behance*
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ]  On the "License" section of your readme, the link is broken because it points to a file that does not exist in your project.

## Tips and extra help

Go [here](https://github.com/microverseinc/readme-template), grab the `MIT.md` file from that repo, paste it on the root of your project, and link to it in the "License" section of your readme.

That way, your app will be well documented 💪